src/portals/hope/features/claims/claim-details/ClaimNotes/ClaimNoteItem.tsx

```diff
@@ -32,20 +33,39 @@ const NoteCreatorInfo = styled.div`
 export const ClaimNoteItem: React.FC<{
   note: ClaimNoteFragment
 }> = ({ note: { text, insertedBy, insertedAt } }) => {
+  // TODO: take type from note data instead
+  const isSystemNote =
+    insertedBy?.__typename === 'UnknownSystemUser' ||
+    insertedBy?.__typename === 'MemberSystemUser'
+  const noteType: 'text' | 'markdown' = isSystemNote ? 'text' : 'markdown'
+  let noteText = <NoteText>{text}</NoteText>
+  if (noteType == 'markdown') {
+    noteText = (
+      <div>
+        <Markdown>{text}</Markdown>
+      </div>
+    )
+  }
+
   return (
     <NoteItem>
-      <NoteText>{text}</NoteText>
+      {noteText}
```

Comment:
nit
I think conditional rendering like the following works best for situations only two options are available ('markdown' or 'text'). But I think it's more like personal preference so either way is fine :)

```suggestion
{noteText === 'markdown'  ? (
  <div>
    <Markdown>{text}</Markdown>
  </div>
) : (
  <NoteText>{text}</NoteText>
)}
```